### PR TITLE
audible-cli: Init at 0.1.3

### PR DESCRIPTION
**Pull request by jvanbruegge**
###### Description of changes

Added the audible package and the audible-cli as new packages

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by jvanbruegge** [comment]
I've updated the PR to the newly released version of the audible package. Still works as expected

**Comment by jvanbruegge** [comment]
I've updated the PR to the newly released version of the audible package. Still works as expected

**Comment by jvanbruegge** [comment]
@jonringer can this be merged?

**Comment by jvanbruegge** [comment]
updated the audible package to version 0.8.1

**Comment by jvanbruegge** [comment]
@jonringer @FRidh @SuperSandro2000 could someone review this? It's been sitting here for a month now

File: pkgs/development/python-modules/audible/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    sha256 = "0fsb5av4s7fvpn0iryl8jj3lwffwlxgbwj46l3fidy0l58nq3b1d";
+  };
+
+  propagatedBuildInputs = [ beautifulsoup4 httpx pbkdf2 pillow pyaes rsa ];
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace 'httpx>=0.20.*,<=0.22.*' 'httpx'
+  '';
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/audible/default.nix:20]:
```suggestion

  pythonImportsCheck = [ "audible" ];

```

Comment (jvanbruegge) [reply pkgs/development/python-modules/audible/default.nix:20]: Added that now

**Pull request merged**